tools/generate-gemm-test.py

```diff
@@ -236,93 +236,52 @@ def split_ukernel_name(name):
         , test_func, isa_check)
         .loop_n(1, nr)
         .loop_m(1, mr));
-  if (k_block > 1) {
```

Comment:
The diff looks messy, but all that actually changed here is that block of tests that are incompatible with the blockwise kernel assumptions are wrapped in the $if KERNELTYPE not in ['qb4w']. I did re-generate all tests via scripts/generate-tests.sh and there are no functional changes to existing tests.